.gitignore

```diff
@@ -6,6 +6,7 @@
 # Build related
 tools/Cake/
 tools/NUnit.Runners/
```

Comment:
Yes, reasoned as if someone used old version they could have that already and repo would be dirty if the pulled down latest.